queries/projects/lup-projects.sql

```diff
@@ -86,15 +97,15 @@ SELECT
         'hearing_location', disp.dcp_publichearinglocation,
         'vote_date', disp.dcp_dateofvote,
         'vote_location', disp.dcp_votelocation,
-        -- 'vote_infavor', disp.dcp_votinginfavorrecommendation,
-        -- 'vote_against', disp.dcp_votingagainstrecommendation,
-        -- 'vote_abstained', disp.dcp_votingabstainingonrecommendation,
-        -- 'vote_total_appointed', disp.dcp_totalmembersappointedtotheboard,
+        'vote_infavor', disp.dcp_votinginfavorrecommendation,
+        'vote_against', disp.dcp_votingagainstrecommendation,
+        'vote_abstained', disp.dcp_votingabstainingonrecommendation,
+        'vote_total_appointed', disp.dcp_totalmembersappointedtotheboard,
```

Comment:
these should be the same as the original column names so things are bit more predictable in the frontend